app/views/user_mailer/invite_email.html.erb

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1>Welcome to example.com,</h1>
+    <p>
+      <%= @organization.name %>
+      <%= @user.email %>
+      You have successfully signed up to example.com,
```

Comment:
Let's write email in Russian?

Reply:
I just prepared email and Masha will do it beautifully